JobsPortal/src/main/java/com/ps/js/controller/JobsController.java

```diff
@@ -0,0 +1,363 @@
+package com.ps.js.controller;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import javax.management.relation.RoleNotFoundException;
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.ps.js.entity.Department;
+import com.ps.js.entity.JobDetails;
+import com.ps.js.entity.JobLocation;
+import com.ps.js.entity.RolesAndResponsebility;
+import com.ps.js.entity.Skill;
+import com.ps.js.exception.ErrorMessages;
+import com.ps.js.exception.JobDetailsNotFoundException;
+import com.ps.js.exception.LocationsNotFoundException;
+import com.ps.js.exception.RolesAndResponsebilityNotFoundException;
+import com.ps.js.exception.SkillNotFoundException;
+import com.ps.js.mapper.JobDetailsMapper;
+import com.ps.js.payload.JobDetailsPayload;
+import com.ps.js.payload.JobDetailsResponsePayload;
+import com.ps.js.payload.JobLocationPayload;
+import com.ps.js.payload.ResponsePayload;
+import com.ps.js.payload.RolesAndResponsebilityPayload;
+import com.ps.js.payload.SkillPayload;
+
+import com.ps.js.service.IDepartmentService;
+import com.ps.js.service.IJobDetailsService;
+import com.ps.js.service.IJobLocationServices;
+import com.ps.js.service.IRolesAndResponsebilityService;
+import com.ps.js.service.ISkillServices;
+
+@RestController
+@RequestMapping("/jobs")
+public class JobsController {
```

Comment:
Even for class also, we need to have java doc comment, check for all other classes as well